docs/contributing.md

```diff
@@ -0,0 +1,31 @@
+For any problems with this charm, please [report bugs here](https://github.com/canonical/content-cache-k8s-operator/issues).
```

Comment:
I'd change the whole content of this file to align with https://github.com/canonical/indico-operator/blob/main/docs/how-to-guides/contributing.md